source/server/admin/clusters_handler.cc

```diff
@@ -45,7 +45,7 @@ ClustersHandler::ClustersHandler(Server::Instance& server) : HandlerContextBase(
 
 Http::Code ClustersHandler::handlerClusters(absl::string_view url,
                                             Http::ResponseHeaderMap& response_headers,
-                                            Buffer::Instance& response, AdminStream&) {
+                                            Server::Chunker& response, AdminStream&) {
```

Comment:
Note that you are already in namespace `Server` so you don't need to qualify `Chunker` here and elsewhere.

Reply:
Get it.